DEVELOPMENT.md

```diff
@@ -6,10 +6,10 @@
 * Download and install [Android Studio] (https://developer.android.com/studio/index.html)
 * Set up [Android Development Environment] (https://spring.io/guides/gs/android/)
 * [Fork this repo] (https://help.github.com/articles/fork-a-repo/)
-* Copy GoIVCodeStyle.xml into the codestyles folder under the IntelliJ home directory (create if it does not exist)
- *  Windows: `\.\config\codestyles\`
- *  OS X: `~/Library/Preferences//codestyles/`
- *  Linux: `~/./config/codestyles/`
+* Copy GoIVCodeStyle.xml into the codestyles folder under the Android Studio config/codestyles directory *(create if it does not exist)*
+ *  Windows: `\%USERPROFILE%\.{ANDROID_STUDIO_FOLDER}\config\codestyles\`
+ *  OS X: `~/Library/Preferences/{ANDROID_STUDIO_FOLDER}/codestyles/`
+ *  Linux: `./{ANDROID_STUDIO_FOLDER}/config/codestyles/`
```

Comment:
I read the commit message. Just saying that docs should be clear on their own, so we should explain what `{ANDROID_STUDIO_FOLDER}` means and give at least an example.
(And right now everybody should be using 2.1, using older versions probably means looking for trouble).